src/include/sof/drivers/ipc.h

```diff
@@ -126,6 +126,17 @@ struct ipc {
 
 extern struct task_ops ipc_task_ops;
 
+static inline uint64_t ipc_task_deadline(void *data)
+{
+	/* TODO: Currently it's a workaround to execute IPC tasks ASAP.
+	 * In the future IPCs should be considered a budget tasks and
+	 * deadline should be calculated based on that assumption.
+	 * This means every IPC should have its own maximum number of
+	 * cycles that is required to finish processing. Based on that
+	 * value we will be able to calculate max deadline allowed.
```

Comment:
This is good thanks. A minor nit, "s/to calculate max deadline allowed/to calculate task deadline/".

Reply:
Done.